core/src/main/scala/filodb.core/query/QueryContext.scala

```diff
@@ -26,10 +24,16 @@ case class PlannerParams(applicationId: String = "filodb",
                          spreadOverride: Option[SpreadProvider] = None,
                          shardOverrides: Option[Seq[Int]] = None,
                          targetSchemaProviderOverride: Option[TargetSchemaProvider] = None,
-                         queryTimeoutMillis: Int = 30000,
-                         sampleLimit: Int = 1000000,
-                         groupByCardLimit: Int = 100000,
-                         joinQueryCardLimit: Int = 100000,
+                         queryTimeoutMillis: Int = GlobalConfig.systemConfig
+                           .getDuration("filodb.query.ask-timeout").toMillis.toInt,
+                         sampleLimit: Int = GlobalConfig.systemConfig
+                           .getInt("filodb.query.sample-limit"),
+                         groupByCardLimit: Int = GlobalConfig.systemConfig
+                           .getInt("filodb.query.group-by-cardinality-limit"),
+                         joinQueryCardLimit: Int = GlobalConfig.systemConfig
+                           .getInt("filodb.query.join-cardinality-limit"),
+                         resultByteLimit: Long = GlobalConfig.systemConfig
+                           .getLong("filodb.query.result-byte-limit"),
```

Comment:
I can change these all back if we want to keep the number of changes minimal.

Reply:
Lets not change this please

Reply:
reverted